astropy/cosmology/core.py

```diff
@@ -60,6 +64,19 @@ class Parameter:
         of the containing Cosmology.
         See https://docs.python.org/3/library/string.html#formatspec
 
+    fixed : bool (optional, keyword-only)
```

Comment:
To avoid the confusion with model parameters, should this have a different name? E.g., `independent=False` or `derived=True`?

Reply:
Hmm. A good point. I'll look at some Cosmology textbooks to see if there's a better nomenclature.